packages/react/src/patterns/blocks/ContentGroupPictograms/__stories__/ContentGroupPictograms.stories.js

```diff
@@ -0,0 +1,142 @@
+import './index.scss';
+import { Desktop, Pattern, Touch } from '@carbon/pictograms-react';
+import { object, select, text, withKnobs } from '@storybook/addon-knobs';
+import classNames from 'classnames';
+import ContentGroupPictograms from '../ContentGroupPictograms';
+import React from 'react';
+import readme from '../README.md';
+import { storiesOf } from '@storybook/react';
+
+storiesOf('Patterns (Blocks)|Content Group Pictograms', module)
```

Comment:
Follow the naming convention. I think everything else is using no spaces.

Reply:
This was applied on the latest commit